JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -50,10 +77,22 @@ module Microsoft.ApplicationInsights.Context {
                 // 365 * 24 * 60 * 60 * 1000 = 31536000000 
                 date.setTime(date.getTime() + 31536000000);
                 var newCookie = [this.id, acqStr];
-                Util.setCookie('ai_user', newCookie.join('|') + ';expires=' + date.toUTCString());
+                Util.setCookie('ai_user', newCookie.join(User.cookieSeparator) + ';expires=' + date.toUTCString());
             }
 
             this.accountId = accountId;
+
+            // get authId and accountId from cookie
+            var authCookie = Util.getCookie('ai_authUser');
+            if (authCookie) {
+                var kvps = authCookie.split(User.cookieSeparator);
+                if (kvps.length > 0) {
+                    this.authId = kvps[0];
+                }
+                if (kvps.length > 1) {
+                    this.accountId = kvps[1];
```

Comment:
The constructor takes account id, but you don't honor it if there is one in cookies. This needs to be clarified